pkgs/applications/misc/homely/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv
+  ,pythonPackages
+}:
+
+pythonPackages.buildPythonPackage rec {
+  version = "0.15.3";
+  pname = "homely";
+
+  #preCheck = ''
+  #  # don't test bash builtins
+  #  rm testing/test_argcomplete.py
+  #'';
+
+  src = pythonPackages.fetchPypi {
+    inherit pname version;
+    sha256 = "2aa7461b583660bf8d9ad776c184fa7a1272141a5c0faaf78a76bc3e8c39db6d";
+  };
+
+  #checkInputs = [ hypothesis ];
+  buildInputs = with pythonPackages; [ attrs setuptools_scm ];
+  propagatedBuildInputs = with pythonPackages; [ requests
+                        simplejson
+                        click
+                        pythondaemon
```

Comment:
As a heads up, I may rename this to its pypi name in https://github.com/NixOS/nixpkgs/pull/47897, pending feedback from @FRidh.

It's cosmetic, but I'd also put requests on the newline to fixup your whitespace a little.